src/game/AnimalManager.ts

```
@@ -466,155 +478,253 @@ export class AnimalManager {
+        ai.stateDuration = 2;
+        ai.velocity = { x: 0, z: 0 };
+        break;
     }
+  }
 
-    const yOffset = animal.config?.yOffset ?? 0.5;
-    animal.position.y = yOffset;
+  /** Trigger FLEE for a specific animal — called by PredatorManager */
+  public frightanAnimal(animalId: string, threatPos: Vec2): void {
+    const animal = this.animals.find((a) => a.id === animalId);
+    if (!animal) return;
```

Biome (2.4.4)

[error] Block statements are preferred in this position.

(lint/style/useBlockStatements)